src/gsy_e/models/area/scm_manager.py

```diff
@@ -310,6 +313,22 @@ def gsy_energy_bill_excl_revenue_without_fees(self):
         return (self.gsy_energy_bill_excl_revenue - self.grid_fees - self.tax_surcharges
                 - self.fixed_fee - self.marketplace_fee - self.assistance_fee)
 
+    @property
+    def gsy_energy_bill_revenue(self):
+        """Energy bill of the home excluding revenue."""
+        return self.earned_from_grid + self.earned_from_community
+
+    @property
+    def gsy_energy_bill_excl_fees(self):
+        """Energy bill of the home excluding revenue."""
+        return (self.gsy_energy_bill - self.grid_fees - self.tax_surcharges
```

Comment:
Can you please remind me of the sign convention of `gsy_energy_bill` ? What if this is negative? Does this mean the user has a credit balance? If yes, the fees should be added here, right?

Reply:
The energy bill includes the fees right from the start (essentially its default value is not 0 but `monthly_fee + assistance_fee + fixed_fee`) therefore the fees should be subtracted from it.